lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -5015,9 +5015,9 @@ LogicalResult StmtEmitter::emitDeclaration(Operation *op) {
 
     // Print debug info.
     if (state.options.printDebugInfo) {
-      StringAttr sym = op->getAttr("inner_sym").dyn_cast_or_null<StringAttr>();
-      if (sym && !sym.getValue().empty())
-        ps << " /* inner_sym: " << PPExtString(sym.getValue()) << " */";
+      auto sym = op->getAttrOfType<hw::InnerSymAttr>("inner_sym");
+      if (sym && !sym.empty())
+        ps << " /* inner_sym: " << PPExtString(sym.getSymName()) << " */";
```

Comment:
`sym.empty()` does not mean there's a symbol on `fieldID == 0`.

Reply:
updated to:
```c++
      auto innerSym = portInfo.at(portIdx).sym;
      if (state.options.printDebugInfo && innerSym && !innerSym.empty()) {
        ps << " /* inner_sym:";
        ps.invokeWithStringOS([&](auto &os) { os << innerSym; });
        ps << " */";
      }
 ```